tests/Behat/Context/Transform/LexicalContext.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file has been created by developers from BitBag.
+ * Feel free to contact us once you face any issues or want to start
+ * You can find more information about us on https://bitbag.io and write us
+ * an email on hello@bitbag.io.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\BitBag\SyliusBonusPointsPlugin\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+
+final class LexicalContext implements Context
```

Comment:
I'm not sure, why you've overridden this class.

Reply:
We need to do that because there was problem in behat test on 1.12 Sylius version. There is function named `getPriceFromString` where we use the @Transform annotation, which varies between versions.